airbyte-config/config-models/src/test/java/io/airbyte/config/helpers/StateMessageHelperTest.java

```diff
@@ -120,4 +123,59 @@ public void testEmptyStateList() {
         .isInstanceOf(IllegalStateException.class);
   }
 
+  @Test
+  public void testLegacyStateConversion() {
```

Comment:
NIT/OPT:
"""
"""
let you create string and multiline string without escaping " and using +. It helps making the json some readable.